mobile_verifier/src/speedtests.rs

```diff
@@ -423,7 +438,7 @@ pub enum FetchError {
 #[async_trait::async_trait]
 pub trait SpeedtestStore {
     async fn fetch(
-        self,
+        &mut self,
```

Comment:
this trait fully deprecated?

Reply:
let me try to bring something like this back for tests